docs/common/yum.md

```diff
@@ -195,6 +189,13 @@ For instructions on upgrading from one OSG series to another, see the
             :::console
             root@host # yum install https://repo.opensciencegrid.org/osg/3.5/osg-3.5-el8-release-latest.rpm
 
+!!! danger "Before considering an upgrade to OSG 3.6&hellip;"
+    OSG 3.6 is under active development and is not currently supported for production use.
+    Due to potentially disruptive changes in protocols, contact your VO(s) to verify that they support token-based
+    authentication and/or HTTP-based data transfer before considering an upgrade to OSG 3.6.
+    If your VO(s) don't support these new protocols or you don't know which protocols your VO(s) support,
+    install or remain on the [OSG 3.5 release series](../release/notes.md)
+
```

Comment:
```suggestion
    !!! danger "Before considering an upgrade to OSG 3.6&hellip;"
        OSG 3.6 is under active development and is not currently supported for production use.

        Due to potentially disruptive changes in protocols, contact your VO(s) to verify that they support token-based
        authentication and/or HTTP-based data transfer before considering an upgrade to OSG 3.6.
        If your VO(s) don't support these new protocols or you don't know which protocols your VO(s) support,
        install or remain on the [OSG 3.5 release series](../release/notes.md)

```

You'll need an additional 4 spaces to pull this into the list item and you need an empty line between the two paragraphs, otherwise it'll just merge it into one